plugins/woocommerce/includes/admin/helper/class-wc-helper-plugin-info.php

```diff
@@ -74,6 +99,11 @@ public static function plugins_api( $response, $action, $args ) {
 		$results = json_decode( wp_remote_retrieve_body( $request ), true );
 		if ( ! empty( $results ) ) {
 			$response = (object) $results;
+
+			$product = array_shift( $products );
+			if ( ! is_null( $product ) && isset( $product['package'] ) ) {
```

Comment:
Do we need both conditions ? If $product is null,` isset( $product['package'] )` will return false anyway. Is this for readability of code ?

```suggestion
			if ( isset( $product['package'] ) ) {
```

Reply:
No, I was just being to defensive. Removed `is_null` in [026b899](https://github.com/woocommerce/woocommerce/pull/40990/commits/026b8992fdc6e92c45b4dfbeee37cafbc21409a4).